packages/amplication-git-utils/src/git/git.service.ts

```diff
@@ -124,6 +125,19 @@ export class GitClientService {
     }
   }
 
+  private async createBranchIfNotExists(args: OneBranchArgs) {
+    const isBranchExist = await this.provider.isBranchExists(args);
```

Comment:
use this.provider.getBranch(args) instead of isBranchExists

Reply:
No because it will throw an error if the branch don't exist